### DISCUSSION TITLE
What do these Housing meta keys in the API mean?

### DISCUSSION THREAD

**Discussion by ixnoahlive** [HypixelDev/PublicAPI#608] [Q&A]
In housingMeta, there's a few odd keys & formattings I've noticed.

- Some users have a YT-Repulsor setting, what does this do?
- What is firstHouseJoinMs? It's not the time you first joined Housing as I've played since 2016.
- ~~In the given_cookies_x option, the x is a number. I assumed this is weeks since 1970 but no. What is the epoch for this? Is it even a week counter?~~ Solved

Edit 1:
- Some users I went thru had coins listed a while ago but these are now removed? Was this a buggy cookie counter?
- Is there a difference between toggle_BORDER and playerSettings.BORDER?

Edit 2:
- Why is given_cookies_x even formatted like that? Why not have a object with every week numbered?

**Comment by mdashlw** [comment]
> * In the given_cookies_x option, the x is a number. I assumed this is weeks since 1970 but no. What is the epoch for this? Is it even a week counter?

see https://github.com/HypixelDev/PublicAPI/issues/303#issuecomment-684174162
